ATS_API/Scripts/Effects/EffectBuilders/CompositeEffectBuilder/CompositeEffectBuilder.cs

```diff
@@ -0,0 +1,230 @@
+﻿using Eremite.Model.Effects;
+using Eremite.Model;
+using System.Collections.Generic;
+using System.Linq;
+using ATS_API.Helpers;
+using System;
+
+namespace ATS_API.Effects;
+
+/// <summary>
+/// This will build <see cref="CompositeEffectModel"/>.
+/// A <see cref="CompositeEffectModel"/> can contain multiple effects.
+/// When players acquire the composite effect, they will receive all the individual effects contained within it.
+/// </summary>
+public class CompositeEffectBuilder : EffectBuilder<CompositeEffectModel>
+{
+    /// <summary>
+    /// Create with the basic information
+    /// </summary>
+    /// <param name="guid">Mod guid</param>
+    /// <param name="name">Effect program name</param>
+    /// <param name="iconPath">icon path</param>
+    public CompositeEffectBuilder(string guid, string name, string iconPath) : base(guid, name, iconPath)
+    {
+        m_effectModel.anyNestedToFit = false;
+        m_effectModel.rewards = [];
+    }
+
+    /// <summary>
+    /// Add an effect to the composite effect
+    /// </summary>
+    /// <param name="effect"></param>
+    public void AddEffect(EffectModel effect)
+    {
+        m_effectModel.rewards.ForceAdd(effect);
```

Comment:
The API works off the assumption core EffectModels do not exist yet so mods can be written in the Awake method of their plugin.
Just realized HookedEffectBuilder also does this which isn't ideal...

Something to look into later is changing this method and others from changing m_effectModel directly but instead doing it in NewEffectData.PostSync and use EffectTypes as parameters instead.